<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>917</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>632</width>
    <height>485</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Replicators</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabs">
      <property name="usesScrollButtons">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1040</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_new_experiment"/>
    <addaction name="action_new_sample_set"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_experiment">
    <property name="title">
     <string>&amp;Experiment</string>
    </property>
    <addaction name="action_refresh_candidates"/>
    <addaction name="action_animate_phenotypes"/>
    <addaction name="action_debug_render"/>
    <addaction name="action_close_tab"/>
   </widget>
   <widget class="QMenu" name="menu_window">
    <property name="title">
     <string>&amp;Window</string>
    </property>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_experiment"/>
   <addaction name="menu_window"/>
  </widget>
  <widget class="QStatusBar" name="status_bar"/>
  <widget class="QToolBar" name="tool_bar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_refresh_candidates"/>
   <addaction name="action_animate_phenotypes"/>
   <addaction name="action_debug_render"/>
  </widget>
  <action name="action_exit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_new_experiment">
   <property name="text">
    <string>&amp;New experiment</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_new_sample_set">
   <property name="text">
    <string>New &amp;sample set</string>
   </property>
  </action>
  <action name="action_refresh_candidates">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Refresh candidates</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_animate_phenotypes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Animate phenotypes</string>
   </property>
  </action>
  <action name="action_debug_render">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Debug render</string>
   </property>
  </action>
  <action name="action_close_tab">
   <property name="text">
    <string>Close tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>action_exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>588</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
